ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml

```diff
@@ -146,6 +149,22 @@ StatusAppTwoPanelLayout {
                         errorDialog.open()
                     }
                 }
+
+                onInviteNewPeopleClicked: {
+                    Global.openPopup(inviteFriendsToCommunityPopup, {
```

Comment:
Where is `inviteFriendsToCommunityPopup` defined? Dynamic scoping is [anipattern](https://github.com/status-im/StatusQ/blob/master/src/contributing.md#scope) we are trying to get rid off, could you please fix that?

Reply:
@osmaczko  That used everywhere, are you sure that I should did it here?

Reply:
Take your popup and give me your approve :D

Reply:
but I think its a good idea with `issue`

Reply:
https://github.com/status-im/status-desktop/issues/6060